widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidget.kt

```diff
@@ -40,7 +41,10 @@ internal class EntryWidgetImpl(
     override fun show(activity: Activity) = activityLauncher.launchEntryWidget(activity)
 
     override fun getView(context: Context): View {
-        val adapter = EntryWidgetAdapter(EntryWidgetContract.ViewType.EMBEDDED_VIEW, themeManager.theme?.entryWidgetTheme)
+        val adapter = EntryWidgetAdapter(
+            Glia.isInitialized(),
```

Comment:
I don't like passing this state directly to the Adapter. What if the Controller handles this?

Reply:
Defining a new `ViewType.UNAUTHENTICATED_ITEM` type may resolve this issue so we can move all the business logic to the controller.

Reply:
@DavDo Yes, we can handle everything [here](https://github.com/salemove/android-sdk-widgets/blob/feature/entry-widget-and-secure-conversations-v2/widgetssdk/src/main/java/com/glia/widgets/entrywidget/EntryWidgetController.kt#L32).

Reply:
Great!

Reply:
@DavDo @AndriiHorishniiMOC Okay. Thank you for the feedback. I introduced a new `ItemType` called `SDK_NOT_INITIALIZED_STATE`. Please have a look now.